language/German/strings.po

```diff
@@ -925,19 +937,24 @@ msgstr "Die Bildschirmdrehung im Cocktailmodus wird nicht unterstützt.\n"
 
 #: src/frontend/mame/ui/info.cpp:135
 msgid "This system requires external artwork files.\n"
-msgstr ""
+msgstr "Dieses System erfordert externe Grafikdateien.\n"
 
 #: src/frontend/mame/ui/info.cpp:137
 msgid ""
 "This system was never completed. It may exhibit strange behavior or missing "
 "elements that are not bugs in the emulation.\n"
 msgstr ""
+"Dieses System wurde nie fertiggestellt. Es kann ein seltsames Verhalten "
+"zeigen oder es können Elemente fehlen, die keine Fehler in der Emulation "
+"sind.\n"
 
 #: src/frontend/mame/ui/info.cpp:139
 msgid ""
 "This system has no sound hardware, MAME will produce no sounds, this is "
 "expected behaviour.\n"
 msgstr ""
+"Dieses System hat keine Sound-Hardware, MAME wird keine Sounds produzieren, "
+"das ist das erwartete Verhalten.\n"
```

Comment:
"das ist erwartetes Verhalten" (without "das"). or "korrektes Verhalten" (translated a bit more freely).

Reply:
Sorry, no.
I'll change the first _das_ to _dies_, but the second _das_ stays